tests/integration/test_rpm.py

```diff
@@ -194,8 +194,8 @@ def read_and_normalize_repofile(path: Path) -> str:
         # RPMs were properly installed
         pytest.param(
             utils.TestParameters(
-                repo="https://github.com/cachito-testing/cachi2-rpm.git",
-                ref="eef99f074a9abd288b421697d7e00cec77590d0d",
+                repo="https://github.com/eskultety/cachito-testing-cachi2-rpm.git",
+                ref="027f3b3068e491e58915ead2cddd775fe380cb4b",
```

Comment:
Note that this currently points to my fork. Based on the acceptance during review I'll proceed with merging the corresponding changes to the upstream counter part.